MkDocs deployment to GH pages not working

GH actions output:

```
INFO    -  Documentation built in 1.04 seconds
WARNING -  Version check skipped: No version specified in previous deployment.
INFO    -  Copying '/home/runner/work/react-ui/react-ui/site' to 'gh-pages' branch and pushing to GitHub.
fatal: could not read Username for 'https://github.com/': No such device or address
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.4/x64/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/mkdocs/__main__.py", line 314, in gh_deploy_command
    gh_deploy.gh_deploy(
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/mkdocs/commands/gh_deploy.py", line 129, in gh_deploy
    ghp_import.ghp_import(
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/ghp_import.py", line 283, in ghp_import
    git.check_call('push', opts['remote'], opts['branch'], '--force')
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/site-packages/ghp_import.py", line 119, in check_call
    sp.check_call(['git'] + list(args), **kwargs)
  File "/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'push', 'origin', 'gh-pages', '--force']' returned non-zero exit status 128.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release-management.yml
+++ b/.github/workflows/release-management.yml
@@ -116,8 +116,6 @@ jobs:
     steps:
       - name: Clone repository
         uses: actions/checkout@v4
-        with:
-          persist-credentials: false
 
       # Hard-coded username and email for the GitHub bot user
       # https://api.github.com/users/github-actions%5Bbot%5D

--- a/src/CNAME
+++ b/src/CNAME
@@ -1,0 +1,1 @@
+react-ui.io

